wezterm-gui/src/overlay/copy.rs

```diff
@@ -1580,22 +1580,34 @@ pub fn copy_key_table() -> KeyTable {
         (
             WKeyCode::Char('c'),
             Modifiers::CTRL,
-            KeyAssignment::CopyMode(CopyModeAssignment::Close),
+            KeyAssignment::Multiple(vec![
```

Comment:
please factor this using a function as I suggested previously

Reply:
I kept this intentionally outside of a helper because a) the documented default config [has inlined Multiples](https://github.com/wez/wezterm/pull/4924/files#diff-aabffc061ef2a12a200c36153cd0551622f2bf61aab2967cdcc6bf11ebc546bbR20), so it felt nice to be consistent, and b) there's one case where we already call `Close` in a `Multiple` ([along with a copy to clipboard](https://github.com/wez/wezterm/pull/4924/files#diff-cefae38b87fd0f76c51a4bc47e630e5bd3a3e4a84821e15c2b9c919f337169edR1861)), so using a helper there would end up with nested `Multiple`s and that felt icky.

I can still add a helper if you prefer, but maybe then I should also add a helper for lua configs? Are nested `Multiple`s ok?

Reply:
I want the rust code to be concise so that it is easier to maintain: I have to live with this code forever. The documented defaults are informational and derived by automation and do not need to have a helper function.

Nested Multiples should be fine.

Reply:
Fair enough, changed.